### ISSUE TITLE
Any plans to update this?

### ISSUE THREAD

**Issue by volkandkaya** [intelligems/django-mobile-app#1]
Docker and following Two Scoops of Django would make this great.

**Comment by koslib** [comment]
Hello there!

Thanks for having a look at our project and I'm glad you find it useful! 

We have plans to keep improving it, and Docker support would actually be a good addition. 
Do you have any other features you'd like to see in here? 

Feel free to either keep posting ideas for improvement or submit a PR!
Thanks!

**Comment by volkandkaya** [comment]
Make it a cookiecutter django project - https://github.com/pydanny/cookiecutter-django

Optimized development and production settings
Add Sentry
Caddy with LetsEncrypt support (HTTPS)
Customizable PostgreSQL version
12-Factor based settings via django-environ

Once that is done the project should explode and you will get a lot more requests.

The last thing would be if you use React Native, would be to create a demo app with authentication and some data loading done. That would save developers so much time.

I'm also going to be using this over the next week so I will see if I can think of anything.

**Comment by koslib** [comment]
Thanks for your recommendations! 

I've already started with some minor updates - better documentation and some unit tests. During the week I'm going to ship more features and updates as well, with Docker-support and production/devel settings being the first two on my list.

In terms of a demo app, we'll create a demo Ember.js project first, to showcase how the API could be consumed in a web dashboard project. Right after that, a React Native demo should be on the way.

Thoughts, feedback or any kind of contribution is widely appreciated!

**Comment by ghost** [comment]
Any update on this? Is there a branch I can contribute to?

**Comment by koslib** [comment]
@CasperCL thanks for reaching out. Currently, we have sort of put it on hold as we do not have many resources to expand this. But you can feel free to checkout `master` and contribute as needed - we'll be there to provide help and push this forward.
